backend/dataall/utils/alarm_service.py

```diff
@@ -72,60 +72,6 @@ def trigger_table_sharing_failure_alarm(
 """
         return self.publish_message_to_alarms_topic(subject, message)
 
-    def trigger_folder_sharing_failure_alarm(
```

Comment:
Shouldn't these two functions also publish a message to the SNS topic like the others?

Reply:
Idk, but they didn't send anything for a long time, so I had a choice to migrate them to datasets/shares and keep them there or just delete. I went with the last one

Reply:
Analyzing the history of the file, this method never triggered an alarm. @dlpzx, do you know anything about this? 
I delete this method here, but will rollback if we understand later why it was done

Reply:
At a first glance, it looks like we are missing `return self.publish_message_to_alarms_topic(subject, message)` at the end of the method. Folder sharing was a feature developed by a collaborator a couple of months ago, let me take a look at the blame